react/features/invite/pendingDtmfMiddleware.js

```diff
@@ -0,0 +1,39 @@
+import { CONNECTED_USER } from '../presence-status';
+
+import { getCurrentConference } from '../base/conference';
+import { StateListenerRegistry } from '../base/redux';
+
+import { storePendingDTMF } from './actions';
+import logger from './logger';
+import { getPendingDtmf } from './selectors';
+
+StateListenerRegistry.register(
+    state => {
+        const jigasiParticipantId
+            = state['features/base/participants']
+                .find(p => p.isJigasi && p.presence?.toLowerCase() === CONNECTED_USER)
+                    ?.id;
+
+        return jigasiParticipantId;
+    },
+    (jigasiParticipantId, { getState, dispatch }) => {
+        const state = getState();
```

Comment:
Is there no phone number associated with the participant? Would it be possible to store pending dtmfs and objects with key as the phone number and dtmf as the value? That way multiple could be stored?

Reply:
As far as I know there is not. But maybe this should be looked into now